packages/office-ui-fabric-react/src/components/Layer/Layer.styles.ts

```diff
@@ -19,6 +19,7 @@ export const getStyles = (props: ILayerStyleProps): ILayerStyles => {
       isNotHost && [
         classNames.rootNoHost,
         {
+          direction: theme.rtl ? 'rtl' : undefined,
```

Comment:
Are you sure this should only apply when `isNotHost` is `true`?

Reply:
oh...totally didn't see that

Reply:
oh funny. I don't actually need this because Layer renders content inside of a Fabric component.

Reply:
What happens for consumers who don't use the `Fabric` component?

Reply:
problem was:

```jsx
<Customizer settings={{theme: RTLTheme}}>
<div dir="rtl">
  <Dropdown .../>
</div>
</Customizer>
```

The above fixes all CSS calculations inside of Dropdown, but all of the layout flipping (flex direction etc) is caused by the `dir="rtl"`. And since the Dropdown options are rendered outside of that div, we need to make sure that Layers pull that RTL from the theme if present. 

And since Layer renders all its contents in a Fabric control, updating Fabric will be enough (i.e. layer style is redundant). 

https://github.com/OfficeDev/office-ui-fabric-react/blob/f6af0479a54f17fd16b9616a62d6be38686a5c1e/packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx#L56-L58